src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1319,10 +1350,20 @@ private IOperation CreateBoundUnconvertedCollectionLiteralExpression(BoundUnconv
             SyntaxNode syntax = boundCollectionLiteralExpression.Syntax;
             ITypeSymbol? collectionType = boundCollectionLiteralExpression.GetPublicTypeSymbol();
             bool isImplicit = boundCollectionLiteralExpression.WasCompilerGenerated;
-            var children = CreateFromArray<BoundExpression, IOperation>(boundCollectionLiteralExpression.Initializers);
+            var children = CreateFromArray<BoundExpression, IOperation>(boundCollectionLiteralExpression.Elements);
             return new InvalidOperation(children, _semanticModel, syntax, collectionType, constantValue: null, isImplicit);
         }
 
+        private IOperation CreateBoundCollectionLiteralSpreadElement(BoundCollectionLiteralSpreadElement boundSpreadExpression)
+        {
+            SyntaxNode syntax = boundSpreadExpression.Syntax;
+            ITypeSymbol? type = boundSpreadExpression.GetPublicTypeSymbol();
+            bool isImplicit = boundSpreadExpression.WasCompilerGenerated;
+            var children = ImmutableArray.Create<IOperation>(Create(boundSpreadExpression.Expression));
+            // PROTOTYPE: Implement:
+            return new InvalidOperation(children, _semanticModel, syntax, type, constantValue: null, isImplicit);
```

Comment:
Prefer `NoneOperation` for unimplemented things please.

Reply:
`ControlFlowGraphBuilder.HandleObjectOrCollectionInitializer()` allows for `Invalid` values but not `None`. I'll leave these as-is for now since they area marked with PROTOTYPE comments to be revisited.